lib/vagrant-openshift/aws.rb

```diff
@@ -44,12 +45,14 @@ def self.fog_config(aws_creds, region="us-east-1")
         }
       end
 
-      def self.find_ami_from_tag(compute, ami_tag_prefix, required_name_tag=nil)
+      def self.find_ami_from_tag(ui, compute, ami_tag_prefix, required_name_tag=nil)
+        ui.info("Searching for latest base AMI")
```

Comment:
I would suggest surfacing the prefix and required name tag, if present, here in this message. I know they were not there originally but they are useful information we'd want if we ever have to debug an AMI selection again.